tests/e2e/resources/installation_config/rds-only.yaml

```diff
@@ -57,7 +57,6 @@ kubeflow-namespace:
     kustomize: 
       - "../../upstream/common/kubeflow-namespace/base"
     helm: "../../charts/common/kubeflow-namespace"
-  namespace: 'kubeflow'
```

Comment:
why?

Reply:
the namespace is for validation purpose only, we are not validating the kubeflow namespace/pod